void __fastcall std::bad_exception::~bad_exception(std::bad_exception *this)
{
  *(_DWORD *)this = &off_468D88;
  nullsub_34();
}
